visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligence.component.scss

```diff
@@ -30,26 +31,77 @@
 	}
 }
 
+.ai-suspicious-metrics-list {
+	display: flex;
+
+	.metric-button {
+		min-height: 36px;
+		line-height: 23px;
+		min-width: 45px;
+		padding: 0 12px 0 12px;
+		text-transform: none;
+		margin-right: 0;
+		overflow: visible;
+	}
+
+	.risk-button {
+		margin-left: 0;
+		min-width: 36px;
+		width: 36px;
+		height: 36px;
+		color: #9c0e6a;
+	}
+}
+
+.ai-drop-down {
+	padding: 0;
+}
+
+md-button.title {
+	color: rgba(0, 0, 0, 0.38);
+	flex-direction: column;
+}
+
+md-menu-item.title-item {
```

Comment:
Maybe give that container a margin to the element below, the space to the next element currently comes from the size of the item, which isnt optimal.